### DISCUSSION TITLE
Releases tab shortens the version number unnecessary

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#43182] [Mobile] [answered]
In the mobile app I'm viewing a reposity that has a release versioned

V4.0.149

But is shows as

V4.0....

![Screenshot_20230103-093212.png](https://user-images.githubusercontent.com/4173387/210322956-e9c3cdbd-5b9a-4c64-b64f-cc89497fb122.png)

**Comment by HenSquared** [comment, marked as answer]
Nice find and thanks for the screenshots.

It looks like this is only impacting the Android app. I'm going to open an issue with the team to get this fixed. Thanks!

**Comment by stevepopovich** [comment]
Hi @jessehouwing. For the life of me, we can not repro this issue, so I am hoping you can answer some questions for me.

1. Are you still see the issue?
2. What version of Android are you using?
3. What version of the app are you using?
4. Are you using any special fonts or font settings?

Thanks!

**Reply by jessehouwing** [reply]
Currently not reproing...
